scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -301,7 +301,11 @@
                                              (when progress
                                                [[:db/add instance :instance/progress progress]])
                                              (when message
-                                               [[:db/add instance :instance/message message]])
+                                               (if (> (count message) executor-message-limit)
+                                                 (log/warn "Executor message too large for task" task-id "as instance" instance
```

Comment:
Truncate to the executor-message-limit.

Also, it might be worthwhile doing the check on the executor side (instead of or in addition to this check) so that we don't have infinity data going over the wire